heat/sparse/_operations.py

```diff
@@ -0,0 +1,160 @@
+"""Generalized MPI operations. i.e. element-wise binary operations"""
+import torch
+import numpy as np
+
+from heat.sparse.dcsr_matrix import DCSR_matrix
+
+from . import factories
+from ..core.communication import MPI
+from ..core.dndarray import DNDarray
+from ..core import types
+
+from typing import Callable, Optional, Dict
+
+__all__ = []
+
+
+def __binary_op_sparse_csr(
```

Comment:
I thought about this. This function/operator only works for CSR matrices. Considering that there might be more sparse formats coming to Heat, I thought it would be a good idea to clearly specify the format for which this operator works.

But maybe we don't have to worry about that now. Let me know if you want me to change it to just `__binary_op`.

Reply:
How about `__binary_op_csr`? I guess we'll end up with `__binary_op_coo` and more

Reply:
Done